### ISSUE TITLE
Property of the Enum type is not included in the Instance JSON

### ISSUE THREAD

**Issue by UlyanaTikhonova** [LIonWeb-org/lioncore-mps#99]
In my small example language, I use MPS enumeration to represent possible binary operations. The concept `BinaryExpression` has a property `operation` of this type. Here is how the language looks as a lioncore language:
![image](https://github.com/LionWeb-io/lionweb-mps/assets/38525290/5ae153f4-7bf5-4bb6-bdb6-e59190f6e760)
Exporting this language into JSON works fine: the enumeration and all its enumeration literals are present (see [ExprLanguageLW.json](https://github.com/user-attachments/files/15987154/ExprLanguageLW.json)). But when I export the simple instance model, the `operation` property of the `BinaryExpression` is not present in the resulting JSON: 
[ExprInstanceLW.json](https://github.com/user-attachments/files/15987162/ExprInstanceLW.json)
Here the instance model is this one:
![image](https://github.com/LionWeb-io/lionweb-mps/assets/38525290/26b599b2-71da-4509-b37b-aac508cd37b2)

**Comment by dslmeinte** [comment]
Hi @UlyanaTikhonova , Is this something you need to have working (correctly) before Linz? (Just for prioritization.)

Probably, the following is just a little bit too simplistic and doesn't take the existence of enumerations into account:

```
project: LionWeb-MPS (/Users/meinteboersma/Documents/LionWeb/lionweb-mps-2021.3)
module:  io.lionweb.mps.json
model:   io.lionweb.mps.json.instance.mps2lionweb
root:    AMps2LionWebConverter [ClassConcept]
node:    getProperty(SProperty):String [InstanceMethodCallOperation]
url:     http://127.0.0.1:63320/node?ref=r%3A42e59ade-052b-4e0d-b0f5-6d4ec03ed4f0%28io.lionweb.mps.json.instance.mps2lionweb%29%2F431394310314209856
```

**Comment by Ulyana-F1re** [comment]
Hi @dslmeinte, yes it would be very nice to have it working, but not critical - as I can change my language to not use enums (it will require changing the Rascal wrapping of the language too). Indeed, it looks like enums require a separate case in converters.

**Comment by dslmeinte** [comment]
@UlyanaTikhonova Thanks for clarifying!

**Comment by enikao** [comment]
Hi @UlyanaTikhonova, Could you check [test2-keyed.json](https://github.com/LionWeb-io/lionweb-mps/pull/110/files#diff-9e1ff11435857b87f16f2d5008f0319a4a72ac3d99f409fb33f77efa58ef4834) on branch `fix-enumliteral-encoding`? I think we _do_ export the literal value (so far with a wrong key, fixed in that PR #110 ):

```
{
      "id": "{id-keyed}",
      "classifier": {
        "language": "NDhkMGY2ZWItNjE4Ni00Y2VjLTgzZDEtN2NhZWRiMDVhNDk0",
        "version": "0",
        "key": "My-Test2ConceptKeyed"
      },
      "properties": [
        {
          "property": {
            "language": "NDhkMGY2ZWItNjE4Ni00Y2VjLTgzZDEtN2NhZWRiMDVhNDk0",
            "version": "0",
            "key": "My-KeyedProp"
          },
          "value": "My-EnumLiteralKeyed"
        },
```

![image](https://github.com/user-attachments/assets/da210315-de6b-4003-9db8-4da617ac53e8)

**Comment by enikao** [comment]
I found the difference between Ulyana's example and my tests: You used enums with default member, and it didn't serialize anything for properties that set that default member value.
Fixed in #110 (including more tests).
